src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/config/redis/RedisConfig.kt

```diff
@@ -0,0 +1,53 @@
+package it.pagopa.ecommerce.eventdispatcher.config.redis
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import it.pagopa.ecommerce.commons.redis.templatewrappers.PaymentRequestInfoRedisTemplateWrapper
+import it.pagopa.ecommerce.commons.redis.templatewrappers.RedisTemplateWrapperBuilder
+import it.pagopa.ecommerce.eventdispatcher.config.redis.bean.ReceiversStatus
+import it.pagopa.ecommerce.eventdispatcher.redis.streams.commands.EventDispatcherReceiverCommand
+import java.time.Duration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.redis.connection.RedisConnectionFactory
+import org.springframework.data.redis.core.RedisTemplate
+import org.springframework.data.redis.serializer.Jackson2JsonRedisSerializer
+import org.springframework.data.redis.serializer.StringRedisSerializer
+
+/** Redis templates wrapper configuration */
+@Configuration
+class RedisConfig {
+  @Bean
+  fun paymentRequestInfoWrapper(
+    redisConnectionFactory: RedisConnectionFactory
+  ): PaymentRequestInfoRedisTemplateWrapper {
+    // PaymentRequestInfo entities will have the same TTL
+    // value
+    return RedisTemplateWrapperBuilder.buildPaymentRequestInfoRedisTemplateWrapper(
+      redisConnectionFactory, Duration.ofMinutes(10))
+  }
+
+  @Bean
+  fun eventDispatcherCommandRedisTemplateWrapper(
+    redisConnectionFactory: RedisConnectionFactory
+  ): EventDispatcherCommandsTemplateWrapper {
+    val redisTemplate = RedisTemplate<String, EventDispatcherReceiverCommand>()
+    redisTemplate.setConnectionFactory(redisConnectionFactory)
+    redisTemplate.setDefaultSerializer(StringRedisSerializer())
+    redisTemplate.afterPropertiesSet()
+    return EventDispatcherCommandsTemplateWrapper(redisTemplate, Duration.ZERO)
```

Comment:
Why zero TTL here?

Reply:
This was set on purpose because this template wrapper is used only to access opsForStreams api (the one used to write events to RedisStreams. No document is written to Redis. Set ttl 0 here raise an error if this is used to write a document to Redis so I set it so.
Add an explanatory comment [here](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/198/commits/3a6d15373b14f70f3cf876cb7e8ba402c4649683)